tasks/0-yagna-setup/README.md

```diff
@@ -16,9 +16,9 @@ Finally, run a verification script to ensure the setup is complete.
 
 ## Steps
 
-1. Follow the yagna daemon setup process available [here](https://docs.golem.network/creators/javascript/quickstart/), until it is running, has funds, and the app key is created.
+1. Follow the yagna daemon setup process available [here](https://docs.golem.network/docs/creators/javascript/quickstart/), until it is running, has funds, and the app key is created.
```

Comment:
broken, needs to be https://docs.golem.network/docs/creators/javascript/quickstarts/quickstart